src/content/blog/release-v1.23.5-auto-port-assignment.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Release v1.23.5: Auto Port Assignment, Auto Timezones"
+pubDate: 2024-10-17
+#modifiedDate: 2024-10-17
+summary: DDEV's v1.23.5 release has support for automatically assigning ports when the default ports are already in use, automatically setting the timezone in the web container, and many other goodies
+author: Randy Fay
+featureImage:
+  src: /img/blog/2024/10/DDEV-v1.23.5.png
+  alt: DDEV v1.23.5 release announcement
+categories:
+  - Announcements
+---
+
+I'm happy to announce that [DDEV v1.23.5](https://github.com/ddev/ddev/releases/tag/v1.23.5) was released today! This has 81 pull requests incorporated into it since v1.23.4, and a number of advances. Every single advance was pushed forward to community needs and community interaction, and we're convinced that our great community is the source of DDEV's strength. Thank you!
+
+**Automatic Port Selection**: In the past, when you started up a DDEV project and the configured ports were already in use, you got an error message. This was [easy to work around](https://ddev.readthedocs.io/en/stable/users/usage/troubleshooting/#web-server-ports-already-occupied), and easy to configure for your system, but it was an unnecessary stop on people's DDEV journey, especially for first-time users. Now, DDEV automatically selects an available port in this situation, and just continues starting your project using the available port. Have apache running? No problem. Have Lando running? No problem. You'll probably want to use the default ports 80 and 443 as you go forward, but there's no reason to stumble starting up a project for the first time. This ambitious and difficult feature was implemented by community member [Alberto Viu](https://github.com/agviu) and me and would never have happened without his initiative and expertise. Thanks!
+
+**Automatic Time Zones**: It's been possible to specify the time zone for the web container for years, but automating it was suggested by community member [Martin Anderson-Clutz (mandclu)
+](https://www.drupal.org/u/mandclu) in the Drupal slack and implemented by Stas. We think that it will "just work" for almost everybody.
```

Comment:
```suggestion
**Automatic Time Zones**: It's been possible to specify the time zone for the web container for years, but automating it was suggested by community member [Martin Anderson-Clutz (mandclu)](https://www.drupal.org/u/mandclu) in the Drupal slack and implemented by Stas. We think that it will "just work" for almost everybody.
```